src/LayerEditor/ui/diagram_editor/graphics_items/gs_point.py

```diff
@@ -114,4 +132,9 @@ def itemChange(self, change, value):
     #
     # def mouseReleaseEvent(self, event):
     #     self.update()
-    #     super().mouseReleaseEvent(event)
\ No newline at end of file
+    #     super().mouseReleaseEvent(event)
+
+    def update_color(self, layer_id: int, regions: RegionsModel):
```

Comment:
Rather extract the color at the calling side and pass just the color to set.

Reply:
This wasn't used anymore and was removed.